build/build.ps1

```diff
@@ -346,9 +354,10 @@ function AddLogCmd([string] $logName, [string[]] $extraArgs)
     Create-Directory $LogDir
     $extraArgs = $extraArgs + ("/bl:" + (Join-Path $LogDir "$logName.binlog"))
 
-    # When running under CI, also create a text log, so it can be viewed in the Jenkins UI
+    # When running under CI, also create a text error log,
+    # so it can be emitted to VSTS.
     if ($ci) {
-      $extraArgs = $extraArgs + ("/flp:Verbosity=diag;LogFile=" + '"' + (Join-Path $LogDir "$logName.log") + '"')
+      $extraArgs = $extraArgs + ("/fileloggerparameters:ErrorsOnly;LogFile=" + '"' + (Join-Path $LogDir "$logName.err") + '"')
```

Comment:
Does `"$logName.err"` actually work? I thought you had to do something like `"$($logName.err)"`

Reply:
Oh, nevermind, `.err` is actually a literal :P

---
In reply to: [210086428](https://github.com/Microsoft/msbuild/pull/3606#discussion_r210086428) [](ancestors = 210086428)